language-plutus-core/src/Language/PlutusCore/Error.hs

```diff
@@ -13,13 +13,10 @@ import           Language.PlutusCore.Renamer
 import           Language.PlutusCore.TypeSynthesis
 import           PlutusPrelude
 
-import           Data.Text as T
```

Comment:
I put this there so I could use it in another PR, please don't delete it again!

Reply:
This is incorrect and should not have been merged without review. Currently the printer for it prints with `"` on both sides.

Reply:
If you'd like to take it out, then do that *in another PR* and we can talk about it there.

Reply:
Sounds good, will do.